src/sentry/debug_files/artifact_bundle_indexing.py

```diff
@@ -179,10 +182,12 @@ def get_flat_file_meta(self) -> Optional[FlatFileMeta]:
         return meta
 
 
-DELETION_KEY_PREFIX = "flat_file_index_deletion"
+DELETION_KEY_PREFIX = "flat_file_index_deletions"
 
 
-def get_deletion_key(flat_file_id: int) -> str:
+def get_deletion_key(flat_file_id: int | None) -> str:
+    if flat_file_id is None:
+        return DELETION_KEY_PREFIX
     return f"{DELETION_KEY_PREFIX}:{flat_file_id}"
```

Comment:
I personally don't like the usage of the same method for two independent keys but if you think it's idiomatic enough, I am good with it.

```suggestion
def get_all_deletions_key() -> str:
    return f"{DELETION_KEY_PREFIX}"
    
def get_deletion_key(flat_file_id: int) -> str:
    return f"{DELETION_KEY_PREFIX}:{flat_file_id}"
```